src/components/ExpandedViewMode/ExpandedViewModeContent/ActivityHistory/ActivityHistory.js

```diff
@@ -75,24 +98,38 @@ class ActivityHistory extends Component {
       <div className='history'>
         {this.differents(this.props.goalHistory).map((value, index) => (
```

Comment:
can you sort the 'history' from "most recent" to "least recent" instead. (it's currently oldest change to newest change)
`this.differents(...).sort(...).map(...`

Reply:
finished